src/base/datetime/time/style.ts

```diff
@@ -84,10 +84,11 @@ export const BASE_TIME_CSS = `
 .kuc-base-time__group--disabled {
   background-color: #d4d7d7;
   box-shadow: none;
-  color: #888888;
-  cursor: not-allowed;
```

Comment:
Don't remove it.
In disabled state, the forbidden icon should be displayed when hovering blank area.